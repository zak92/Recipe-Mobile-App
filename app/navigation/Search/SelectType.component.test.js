
import React from 'react';
import { render } from '@testing-library/react-native'; // json rendering
import SelectType from './SelectType.component';


describe('SelectType', () => {
  it("should match snapshot", () => {
    const snapSelectType = render(<SelectType />).toJSON(); // render this file to JSON
    expect(snapSelectType).toMatchSnapshot(); // this snapshot should match the previous snapshot

  });
  it('displays the passed-in dishType', () => {
    const { queryByText } = render(
      <SelectType type="french" />
    );
    expect(queryByText('french')).not.toBeNull();
  });


});