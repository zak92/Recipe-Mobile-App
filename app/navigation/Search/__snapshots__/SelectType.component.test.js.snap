// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectType should match snapshot 1`] = `
<View
  style={
    Object {
      "color": "black",
      "padding": "5%",
      "textAlign": "left",
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
          "fontFamily": "Roboto_400Regular",
          "fontSize": 16,
          "textAlign": "left",
        }
      }
      testID="type"
    />
  </View>
</View>
`;
